elements/pfe-jump-links/pfe-jump-links.ts

```diff
@@ -1,17 +1,81 @@
-import { PfeJumpLinksNav } from './pfe-jump-links-nav.js';
-export { PfeJumpLinksPanel } from './pfe-jump-links-panel.js';
-export { PfeJumpLinksNav };
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
 
-import { pfelement } from '@patternfly/pfe-core/decorators.js';
-import { customElement } from 'lit/decorators.js';
+import { ScrollSpyController } from '@patternfly/pfe-core/controllers/scroll-spy-controller.js';
+
+import './pfe-jump-links-item.js';
+import '@patternfly/pfe-icon';
+
+import style from './pfe-jump-links.scss';
 
 /**
- * Jump links act as persistent navigation that consists of a list of anchor links.
- * Selecting a link moves a user to content that corresponds with the link selected.
- * A link is displayed as active when the content it links to is visible in the browser window.
- *
- * @summary Moves a user to content when a link is selected
- * @cssprop --pfe-jump-links-panel--offset You can control offset in your styling layer as well. This value can be set directly on the component inside a style attribute, e.g. style="--pfe-jump-links-panel--offset: 100;" or using the appropriate selector in another file. Please note that adding an attribute will take precedence over a css value. At the moment only integer values passed to this key are valid. No other values are supported. This means that passing "300px", "2rem","calc(100% - 12px)" will all result in JavaScript errors. You should pass a number that correlates to pixels. To read about the offset attribute, see above.
+ * **Jump links** allow users to navigate to sections within a page.
+ * @fires toggle - when the `expanded` disclosure widget is toggled
+ * @slot - Place pfe-jump-links-items here
```

Comment:
Do we want to document the PF CSS props here?

Reply:
Yes, we should